ansible/group_vars/all

```diff
@@ -24,18 +26,25 @@ nginx_vhosts:
     return: "302 http://$hostname"
   - listen: "80"
     server_name: "$hostname"
-    index: "index.php index.htm index.html"
-    root: "/var/www/biblebox"
-    error_page: "404 /biblebox_default/content/index.html"
+    index: index.html
+    root: "{{ biblebox_default_content_root }}"
+    error_page: "404 /index.html"
     access_log: "/var/log/nginx/biblebox-access.log"
     error_log: "/var/log/nginx/biblebox-error.log"
     extra_parameters: |
       rewrite_log on;
 
-      # pass the PHP scripts to FastCGI server listening on 127.0.0.1:9000
+      # Admin interface
+      location /admin/ {
+        root {{ biblebox_web_root }};
```

Comment:
Only if we had `alias {{ biblebox_admin_root }};` instead. Given `/admin` is a common suffix, I'm told it's more idiomatic to use a `root` element, even though they're functionally identical.